planetiler/src/main/java/com/onthegomap/planetiler/examples/LoliStyle.java

```diff
@@ -0,0 +1,128 @@
+package com.onthegomap.planetiler.examples;
```

Comment:
We could switch to Kotlin for this stuff. I'm not sure how to configure it with maven, but with Gradle it is about 5 lines.

To be honest, I'm not sure how latest Java and Kotlin compare.

Reply:
Let's stick to java for now. Switching should not be too hard if we want to go to Kotlin one day.